realtime/redisSubscriber.js

```diff
@@ -11,7 +11,7 @@
  */
 'use strict'; // eslint-disable-line strict
 const emitter = require('./socketIOEmitter');
-const sub = require('../cache/redisCache').client.sub;
+const subPerspective = require('../cache/redisCache').client.subPerspective;
```

Comment:
is perspective a good work because refocus already uses that word to mean something else?

Reply:
that was ians suggestion he wanted to avoid it being refocus/imc

Reply:
Okay cool, long as it got permission then sounds good